Automattic/woocommerce-payments
# Add E2E tests for card testing mitigation measures

Fixes #8163

#### Changes proposed in this Pull Request


This PR enhances existing checkout tests to include the payment attempts with the protection on, and the fraud prevention token checking is in place.







#### Testing instructions

> [!IMPORTANT]
> This needs to be tested after #8142 is merged.





* Prepare your local E2E environment using `npm run test:e2e-reset && npm run test:e2e-setup`. More details can be found in the `tests/e2e/README.md` file. 
* After your local environment is ready, run the tests with `npm run test:e2e` 
* All tests should pass.

-------------------


**Post merge**
